<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SravnenieWindow</class>
 <widget class="QWidget" name="SravnenieWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButtonSavePDF">
     <property name="text">
      <string>Сохранить в PDF</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QCustomPlot" name="widget" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Диски</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonStartSravn">
     <property name="text">
      <string>Запустить все</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>Процессы</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>Подкачка</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonStopSravn">
     <property name="text">
      <string>Остановить все</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>ОЗУ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButtonSaveJPG">
     <property name="text">
      <string>Сохранить в jpg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Добавить к сравнению:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Закрыть окно</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="text">
      <string>Добавить из log-файла</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>Сеть - отправка</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>Сеть - прием</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
